pkg/katautils/config.go

```diff
@@ -356,6 +352,12 @@ func (h hypervisor) getInitrdAndImage() (initrd string, image string, err error)
 		return "", "", errors.New("either image or initrd must be defined in the configuration file")
 	}
 
+	if initrd != "" && errInitrd != nil {
```

Comment:
We still err out when initrd is not installed?

Reply:
If initrd is specified in the toml file but that file does not exist, then we error out.

Reply:
Original Code: If I just install initrd and then run Kata, it errors out stating that path/to/rootfs.image does not exist. 

In this patch: I  check just one of the two,  initrd OR image path are specified(uncommented) in the configuration.toml and then check the one that was specified, that file actually exist.

Reply:
`ResolvePath()` returns either a valid path with nil error, or empty path with an error. So we never get to the point here that `initrd` is not empty while `errInitrd` is non-nil, nor the line below that `image` is not empty while `errImage` is set.

So to make the code work, I think you only need to make sure `errInitrd` and `errImage` are not both set.

Reply:
Thanks @bergwolf